src/app/debug/table/table.component.ts

```diff
@@ -437,19 +450,39 @@ export class TableComponent implements OnInit, OnDestroy {
     return fullName;
   }
 
-  getTableSpacing() {
+  getTableSpacing(): string {
     return `${this.tableSpacing * 0.25}em 0 ${this.tableSpacing * 0.25}em 0`;
   }
 
-  getFontSize() {
-    return `${8 + this.tableSpacing * 1.2}pt`;
+  getFontSize(): string {
+    return `${this.defaultFontSize + this.tableSpacing * this.fontSizeSpacingModifier}pt`;
+  }
+
+  getCheckBoxSize(): string {
+    return `${this.defaultCheckBoxSize + this.tableSpacing}px`;
   }
 
-  getCheckBoxSize() {
-    return `${13 + this.tableSpacing}px`;
+  setUniqueOptions(data: any): void {
```

Comment:
Why is data `any`?

Reply:
because theres no type for this data structure/model